spot_rl_experiments/spot_rl/envs/gaze_env.py

```diff
@@ -197,11 +232,34 @@ def step(self, base_action=None, arm_action=None, grasp=False, place=False):
         grasp = self.should_grasp()
 
         observations, reward, done, info = super().step(
-            base_action, arm_action, grasp, place
+            base_action,
+            arm_action,
+            grasp,
+            place,
+            max_joint_movement_key="MAX_JOINT_MOVEMENT_MOBILE_GAZE"
+            if self._use_mobile_pick
+            else "MAX_JOINT_MOVEMENT",
+            max_lin_dist_mobile_gaze="MAX_LIN_DIST_MOBILE_GAZE"
+            if self._use_mobile_pick
+            else None,
+            max_ang_dist_mobile_gaze="MAX_ANG_DIST_MOBILE_GAZE"
+            if self._use_mobile_pick
+            else None,
```

Comment:
Let's clean the function args @jimmytyyang

Reply:
okay, fixed this! thank you!